File: abc/abc287/a.cpp
// https://atcoder.jp/contests/abc287/submissions/38375928

#include <bits/stdc++.h>
using namespace std;

int main() {
  int n;
  cin >> n;

  int cnt = 0;
  string s;
  while (cin >> s) {
    if (s == "For") cnt++;
  }

  if (n / 2 < cnt)
    cout << "Yes";
  else
    cout << "No";
}

File: abc/abc287/a.cs
// https://atcoder.jp/contests/abc287/submissions/46509290

using System;

public class A
{
  static void Main()
  {
    int n = int.Parse(Console.ReadLine());
    int cnt = 0;

    for(int i = 0; i < n; i++)
    {
      if(Console.ReadLine() == "For")
      {
        cnt++;
      }
    }

    double h = n / 2.0;
    if(cnt >= h)
    {
      Console.Write("Yes");
    }
    else
    {
      Console.Write("No");
    }
  }
}

